src/core/renderers/SystemRenderer.js

```diff
@@ -69,20 +69,13 @@ export default class SystemRenderer extends EventEmitter
         this.type = RENDERER_TYPE.UNKNOWN;
 
         /**
-         * The width of the canvas view
+         * Measurements of the screen. (0, 0, screenWidth, screenHeight)
          *
-         * @member {number}
-         * @default 800
-         */
-        this.width = width || 800;
-
-        /**
-         * The height of the canvas view
+         * Its safe to use as filterArea or hitArea for whole stage
          *
-         * @member {number}
-         * @default 600
+         * @member {PIXI.Rectangle}
          */
-        this.height = height || 600;
+        this.screen = new Rectangle(0, 0, screenWidth || 800, screenHeight || 600);
```

Comment:
Could you make sure that you set `this.screen` to null when the SystemRenderer is destroyed?